services/app-api/src/postgres/oauth/oauthClientStore.ts

```diff
@@ -28,6 +32,7 @@ export async function createOAuthClient(
                 grants,
                 description: data.description,
                 contactEmail: data.contactEmail,
+                userID: data.userID,
             },
```

Comment:
Lets include the user data in the return.

```suggestion
            },
            include: {
               user: true
            }
```